### DISCUSSION TITLE
,.

### DISCUSSION THREAD

**Discussion by Nishmitha24** [github/india#346] [Students]
is completing self learning mandatory?

**Comment by manavmodi22** [comment]
Yes.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/jPv7hydk

Thank you 🙏
